internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -586,16 +586,14 @@ func convertCommand(command *manifest.CommandOverride) ([]string, error) {
 	return out, nil
 }
 
-func convertPublish(p *manifest.PublishConfig, ta map[string]string) (*template.PublishOpts, error) {
-	if p == nil || len(p.Topics) == 0 || ta == nil {
+func convertPublish(p *manifest.PublishConfig, accountID string, region string, app string, env string, svc string) (*template.PublishOpts, error) {
```

Comment:
Go has a nifty bit of syntactic sugar
```suggestion
func convertPublish(p *manifest.PublishConfig, accountID, region, app, env, svc string) (*template.PublishOpts, error) {
```

Reply:
Neat! Adjusted all the lengthy string functions.